packages/wrap/tests/transforms.test.ts

```diff
@@ -1487,3 +1488,69 @@ describe('replaces field with processed fragment node', () => {
     expect(result.data.test.field2).toBe('field2');
   });
 });
+
+describe('transform TransformEnumValues', () => {
```

Comment:
suggestion: Would it make sense to also add a test for changing the `value` of an GraphQLEnumValue?

If I'm grokking the code correctly, outside of renaming an existing value this transform would also let the user modify the "internal value" associated with an enum value without necessarily changing the name. It might be a good idea to add a test for that as well.

Reply:
You can actually do that already without this transform simply by adding the enum values to the resolver map within stitchSchemas just as within makeExecutableSchema. Although you can change the internal value via this transform also, it's not the easiest way to do it.

I believe we test the easier approach, but can't find it at the moment. :(

Reply:
But makes sense to add test that works in this manner anyway, good idea!